extensions/wikia/Helios/HelperController.class.php

```diff
@@ -52,7 +56,7 @@ public function updateAntiSpoof()
 
 		if ( !empty( $this->wg->EnableAntiSpoofExt ) ) {
 			$oSpoofUser = new \SpoofUser( $sName );
-			$oSpoofUser->record();
+			return $this->response->setVal( 'success', $oSpoofUser->record() );
```

Comment:
My plan is to return the result here to know what was the operation outcome and to fix SpoofUser::record() implementation.